react/utils/IOMessageWithMarkers.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/display-name */
+import React, { FC } from 'react'
+import { IOMessage } from 'vtex.native-types'
+import { useCssHandles } from 'vtex.css-handles'
+
+interface Props {
+  label: string
+  componentName: string
+  markers: string[]
+  rest: any
+}
+
+const IOMessageWithMarkers: FC<Props> = ({
+  label,
+  markers = [],
+  componentName,
+  rest,
```

Comment:
What is rest?

Reply:
This would only work with `...rest`

Reply:
But I'm not a big fan of this name. Maybe `additionalProps`?

Reply:
Going to rename to `values`